.firebase/hosting.c3RhdGlj.cache

```diff
@@ -0,0 +1,9 @@
+404.html,1585258280839,3240c6b45513fdd97861a63c8ae3ef0e7d2bf1357ec632ad7e6839473710326c
+favicon.png,1584503083807,4e81307d839c59f6b2d3548ad4ec33c016a699849aa39c4c2e10501d583d84f2
+great-success.png,1584503083810,805d0642b24ae837193f44a87b084893039f7ae7e74f5ec817d0df9658470652
+logo-192.png,1584503083812,6acc2b42f7d34eaa4c016bf29427bfee481dda57afcfa62fe106c664f57e3ed2
+logo-512.png,1584503083813,d138395999f271cfabc5cc27dc59beab0a98034089beb693868b075973a126ac
+manifest.json,1584503083814,4f61ce58c68fc2d8b3df7908d1ed2857a5e596dbecb3f87e4fb2f426955bf62a
+images/home-image.svg,1585251507762,68c85c8554c42cc430b8d5b4d67a2899387cc47712675a2d9a242b85a9b390bf
+images/home-page-image.png,1585251507763,8fb9269a83c9d8e53a8d579b363142b225febe1b2fc6384514332de807ddc3a3
+global.css,1585336682741,51d443bad4b19dc48e2cef55038af2137bdfe0966ebc22c667f6668eb772f375
```

Comment:
Looks like you're not supposed to check in this file according to your `.gitignore`. Make sure to uncommit this file :)